CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 This is a major release, and it might be not compatible with your current usage of our library. Please read about the necessary changes in the section about how to migrate.
 
+### Migration from v24 to v25
+
+-   remove deprecated components, properties and imports from your project, if the info cannot be found here then it was already mentioned in **Deprecated** sections of the past changelogs
+-   in case you set your own colors before importing GUI elements you need to update your configuration to the new color palette structure, see `README.md`
+-   change `intent="primary"` to `intent="accent"` for `<Button />`, `<IconButton />` and `<Spinner />`, if supported it would be even better to use `affirmative={true}` or `elevated={true}` instead of `intent` in morst cases
```

Comment:
typo 'morst' -> most

Since the `affirmative` or `elevated` properties are supported on these three components, why even recommend changing it to "accent". The text sounds that the other options are preferable.